domains/certificates/docker-test-env/docker-compose.yml

```diff
@@ -16,6 +16,17 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
 
+  registry:
+    image: ghcr.io/project-origin/electricity-server:0.1.0-alpha.18
+    ports:
+      - 8765:80
+    environment:
+      IMMUTABLELOG__TYPE: "log"
+      REGISTRIES__RegistryA__VERIFIABLEEVENTSTORE__EVENTSTORE__TYPE: "inMemory"
```

Comment:
Should all thse env vars not be upper case?

Reply:
It is not a rule, but a convention. I choose to write it as "RegistryA" over "REGISTRYA" to make it easier to read.

Reply:
I have also tested to see if the registry is case sensitive wrt. the registry name. And it is case sensitive in the current version. So "RegistryA" is not the same as "REGISTRYA" or "registryA" or "registrya".
Since I would like for the final name to be "RegistryA", then I would like to keep this